### DISCUSSION TITLE
Keep multiple versions

### DISCUSSION THREAD

**Discussion by VilCoyote** [enzo1982/freac#301] [Help & Support]
Hello,
Is it possible to configure the software to keep songs that have several versions?
Thank you

**Comment by enzo1982** [comment]
I'm not sure what's the issue here. fre:ac does not care what songs it converts, so of course you can have several versions of one.

I'm guessing though that the issue is with how fre:ac names output files. For example, if you have the following input files:

```
Some Artist - Some Song.flac
Some Artist - Some Song ver 2.flac
```

And the metadata in both files just has _Some Song_ as the title (i.e. the second one does not mention _ver 2_ in its `TITLE` tag), then by default, both files may get the same output filename.

If that is the issue, then the solution is to configure a different output filename pattern in fre:ac's configuration. The default output filename pattern is: `<albumartist> - <album>\<track> - <artist> - <title>`

You can set the output filename pattern to `<filename>` to keep the original file names, thus avoiding the files getting the same name. Alternatively, there is an option to append a sequential number to otherwise identical file names.

You can find both options at the bottom of the initial page after opening the config dialog.

**Reply by VilCoyote** [reply]
Hello and thank you very much for your quick response.
I will implement your advice and will keep you informed.
Sincerely, Alain
